LeaderboardBackend/Models/Requests/LeaderboardRequests.cs

```diff
@@ -1,8 +1,11 @@
+using System.ComponentModel.DataAnnotations;
 using FluentValidation;
 using LeaderboardBackend.Models.Validation;
 
 namespace LeaderboardBackend.Models.Requests;
 
+#nullable disable warnings
```

Comment:
Okay so if I understand this line correctly; we're disabling the compiler warnings that the DTO fields may be nullable, because we _can_ expect them to be, right?

Reply:
Yes. We want to allow the props to implicitly default to null but to then fail validation. Otherwise it fails to bind at all with missing props and returns 400. It's a bit gross but so is the way c# does nullability in general.